homeassistant/components/spider/climate.py

```diff
@@ -11,9 +11,8 @@
 
 from .const import DOMAIN
 
-SUPPORT_FAN = ["Auto", "Low", "Medium", "High", "Boost 10", "Boost 20", "Boost 30"]
-
-SUPPORT_HVAC = [HVAC_MODE_HEAT, HVAC_MODE_COOL]
+SUPPORT_FAN = []
+SUPPORT_HVAC = []
```

Comment:
If these values are not constants, they should not be defined as constant. Move these into a class variable instead.

Reply:
Done. See https://github.com/home-assistant/core/pull/44575/commits/7b29c1f1a8ff5cd423b0d9cd8978a183c9564416